packages/block-editor/src/components/tabbed-sidebar/style.scss

```diff
@@ -32,4 +32,10 @@
 	flex-direction: column;
 	overflow-y: auto;
 	scrollbar-gutter: auto;
+
+	// Use `position: relative` to ensure any absolute positioned child elements are
+	// positioned relative to the tab panel.
+	// This makes the overflow rule of the panel work correctly, particularly when the
+	// `VisuallyHidden` component is used within the inserter UI.
+	position: relative;
```

Comment:
I think this has caused a regression on the patterns preview panel.

<img width="592" alt="Screenshot 2024-10-21 at 10 40 05 am" src="https://github.com/user-attachments/assets/215acdb1-d0c0-45c6-923e-3a880f9f02fd">

Reply:
Update: this regression was fixed in https://github.com/WordPress/gutenberg/pull/66267. Thanks @tellthemachines for landing it!